pom.xml

```diff
@@ -1,14 +1,14 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+    <modelVersion>4.3.0</modelVersion>
```

Comment:
It is an error to change the modelVersion value. It is a value that is specific to Maven, not Openfire. Please keep this at 4.0.0.

Reply:
I think i have fixed it